runtime/src/main/java/org/corfudb/runtime/clients/LogUnitClient.java

```diff
@@ -79,6 +81,23 @@ public Integer getPort() {
     public ClientMsgHandler msgHandler = new ClientMsgHandler(this)
             .generateHandlers(MethodHandles.lookup(), this);
 
+    /** Handles the LOGUNIT_METRICS_RESPONSE
+     *
+     * @param msg   Incoming message
+     * @param ctx   Context
+     * @param r     Router
+     * @return      byte[] of the LogUnitMetrics object.
+     */
+    @ClientHandler(type=CorfuMsgType.LOGUNIT_METRICS_RESPONSE)
+    private static Object handleMetricsResponse(CorfuPayloadMsg<byte[]> msg,
+                                                ChannelHandlerContext ctx, IClientRouter r) {
+        try {
+            return LogUnitMetrics.parseFrom(msg.getPayload());
+        } catch (InvalidProtocolBufferException e) {
+            throw new RuntimeException(e);
```

Comment:
Not sure if RuntimeException is the right thing here. We need to log somewhere if protocol buffers are corrupt.